src/db/models/BooklistedToken.js

```diff
@@ -0,0 +1,14 @@
+export default (sequelize, DataTypes) => {
+  const BooklistedToken = sequelize.define('BooklistedToken', {
```

Comment:
```suggestion
  const BooklistedToken = sequelize.define('BooklistedTokens', {
```
I think this should be `Blacklisted` not `Booklisted`

Reply:
Yes, done